pkg/mounter/mounter-s3fs.go

```diff
@@ -147,8 +147,12 @@ func (s3fs *S3fsMounter) Mount(source string, target string) error {
 		return fmt.Errorf("S3FSMounter Mount: Cannot create file %s: %v", passwdFile, err)
 	}
 
-	if s3fs.ObjPath != "" {
-		bucketName = fmt.Sprintf("%s:/%s", s3fs.BucketName, s3fs.ObjPath)
+	if s3fs.objectPath != "" {
+		path := strings.TrimPrefix(s3fs.objectPath, "/")
+		if path != "" {
+			path = "/" + path
+		}
+		bucketName = fmt.Sprintf("%s:%s", s3fs.BucketName, path)
```

Comment:
Refer following code https://github.com/IBM/ibmcloud-object-storage-plugin/blob/master/driver/driver.go#L581-L589 to handle this condition:
```
if options.ObjectPath != "" {
		if strings.HasPrefix(options.ObjectPath, "/") {
			fullBucketPath = options.Bucket + ":" + options.ObjectPath
		} else {
			fullBucketPath = options.Bucket + ":/" + options.ObjectPath
		}
	} else {
		fullBucketPath = options.Bucket
	}
```